src/services/twilio.ts

```diff
@@ -144,6 +145,7 @@ export const KnownTwilioErrorMessages: { [key: string]: RegExp } = {
   InvalidMobileNumber: /not a mobile number/,
   InvalidPhoneNumberParameter: /Invalid parameter `To`/,
   RestrictedRegion: /has not been enabled for the region/,
+  RestrictedCountryChina: /China require use case vetting/,
```

Comment:
can we just validate against `require use case vetting` ? having an error type for each country is not an option (is not only china)

Reply:
agree I would not make it country specific. we don't know which other country twilio requires "vetting"